packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -119,10 +119,11 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
 
         if (window.__swc.DEBUG) {
             if (
-                !this.label &&
-                !this.getAttribute('aria-label') &&
-                !this.getAttribute('aria-labelledby') &&
-                !this.slotEl.assignedNodes().length
+                (!this.label ||
```

Comment:
I needed to re-read this a couple of times 😛 If I got it right, we are saying "if this element is of role progressbar but it has no text label, no aria-label, no aria-labelledby, and no slotted label content, then we throw the warning".

What do you think of making it a bit more clear (pseudo-code):

```ts
hasAccessibleName() {
    return Boolean(
        this.label ||
        this.getAttribute('aria-label') ||
        this.getAttribute('aria-labelledby') ||
        this.slotEl.assignedNodes().length
    );
}
```

and then

```ts
if (!this.hasAccessibleName() && this.getAttribute('role') === 'progressbar') {
```